### PR TITLE
Feat/user onboard

### PR DESCRIPTION
**Pull request by Aditi0505**
# Change Logs:

## Description:

- Added tailwind configuration and custom height property
- Added User Onboarding Page
- Added User Welcome Page
- The wallpaper changes on refresh
- The user details persist on refresh
- The user name can be edited
- The focus can be marked completed, can be edited, or deleted

## Screenshot 
image
image

## Files to be ignored while reviewing

-  wallpaper images
- package.json
- package.lock.json

## Preview Link:
- -

### CODE REVIEW

**Review by aniketsingh13 — APPROVED** [review]
UI look good
code is easy to read
good to merge

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,19 @@
+import { Route, Routes } from "react-router-dom";
+import { UserOnboarding, WelcomeUser } from "./pages";
 import "./App.css";
-
+import { WallpaperProvider } from "./context/wallpaper-context";
```
Comment (ranitasahaa1230) [inline src/App.js:4]: import `WallpaperProvider` from context only, no need for line 7 then

**Review by ranitasahaa1230 — APPROVED** [review]
Good use of reducers and contexts
Good work with the tailwind
Code looks clean,
Looks good to merge

File: src/App.js (hunk 1 above)
Comment (Aditi0505) [reply src/App.js]: Thank you for pointing this out. I have fixed this.

**Pull request merged**